.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,6 +22,7 @@ After:
 - [ ] **DONE**
 
 ## Test coverage
+ℹ️ If a major new functionality is added, tests for the new functionality MUST be added to the Cucumber test suite
```

Comment:
IMHO `MUST` is not really encouragement, but enforcement.

Which one do we want? I'd rather go with encouragement for now, so maybe:
```
If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite
```
?

Reply:
Ok. If you ask me, I would enforce it, for major new functionality :)
But ok, let's start with a bit less ambition hehe